javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,143 @@
+             /********************************************************/
+               console.log('\n        Smallest letter in Array\n\n')
+             /********************************************************/      
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const smalleLetter=getSmallestLetter(danishWords); // returns 'ø'
+function getSmallestLetter(words){
+    let temp=danishWords[0];
+    for(let i=0;i<danishWords.length;i++){
+         if(danishWords[i].length<=temp.length){
+             temp=danishWords[i];
+             danishWords[i]=danishWords[0];
+             danishWords[0]=temp;
+
```

Comment:
why are you assigning the variable so weirdly here? Dont you only need `temp = danishWords[i]` and later `return temp`

Reply:
done